Resources/views/macros/data-field-type.html.twig

```diff
@@ -637,6 +637,35 @@
 	{{ _self.messages(dataField) }}
 {% endblock %}
 
+
+
+{% block EMS_CoreBundle_Form_DataField_JsonMenuEditorFieldType %}
+	{% set doCompare = compare and dataField.rawData is not same as( compareRawData|get_string(dataField.fieldType.name) )  %}
```

Comment:
DoCompare seems business logic for PhP side.

Reply:
For once I disagree, It's the view to compare fields between two revision